augraphy/augmentations/markup.py

```diff
@@ -115,51 +115,43 @@ def _preprocess(self, image):
 
         return dilation
 
-    def draw_line(self, p1, p2, p1_x, p1_y, p2_x, p2_y, xsize, ysize, markup_mask, markup_thickness):
-        # draw main line
-        markup_mask = cv2.line(
-            markup_mask,
-            p1,
-            p2,
-            self.markup_color,
-            markup_thickness,
-            lineType=cv2.LINE_AA,
-        )
+    def draw_line(self, p1, p2, markup_mask, markup_thickness, reverse):
+
+        # get min and max of points
+        min_x = min(p2[0], p1[0])
+        max_x = max(p2[0], p1[0])
+        min_y = min(p2[1], p1[1])
+        max_y = max(p2[1], p1[1])
 
-        # draw another line with offset y at point1 or point2, so that one end of the line is thicker, another end is thinner
-        if random.random() > 0.5:
-            p1_y_offset1 = np.clip(p1_y + markup_thickness, 0, ysize)
-            p1_y_offset2 = np.clip(p1_y - markup_thickness, 0, ysize)
-            p11 = (p1_x, p1_y_offset1)
-            p21 = (p2_x, p2_y)
-            p12 = (p1_x, p1_y_offset2)
-            p22 = (p2_x, p2_y)
+        # set point x in ascending or descending order based on direction
+        if reverse:
+            points_x = [min_x, random.randint(min_x, max_x), max_x]
         else:
-            p2_y_offset1 = np.clip(p2_y + markup_thickness, 0, ysize)
-            p2_y_offset2 = np.clip(p2_y - markup_thickness, 0, ysize)
-            p11 = (p1_x, p1_y)
-            p21 = (p2_x, p2_y_offset1)
-            p12 = (p1_x, p1_y)
-            p22 = (p2_x, p2_y_offset2)
-
-        # draw positive offset
-        markup_mask = cv2.line(
-            markup_mask,
-            p11,
-            p21,
-            self.markup_color,
-            markup_thickness,
-            lineType=cv2.LINE_AA,
-        )
-        # draw negative offset
-        markup_mask = cv2.line(
-            markup_mask,
-            p12,
-            p22,
-            self.markup_color,
-            markup_thickness,
-            lineType=cv2.LINE_AA,
-        )
+            points_x = [max_x, random.randint(min_x, max_x), min_x]
```

Comment:
With this update, now markup crossed effect is using curvy lines, for example:

![image](https://user-images.githubusercontent.com/52521979/159168930-3a3d32f5-8a31-4736-a761-12982a8eae63.png)

![image](https://user-images.githubusercontent.com/52521979/159168937-6ea640c4-0ee3-46d9-9d77-a43245a91d52.png)

![image](https://user-images.githubusercontent.com/52521979/159168943-b594dc30-e1c4-4ea7-87d1-89b1b47ccad2.png)